WordPress/Classes/Stores/StatsInsightsCache.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+final class StatsInsightsCache {
+    static let shared = StatsInsightsCache()
+
+    private var cache: [CacheKey: Any] = [:]
```

Comment:
Based on the `getValue` and `setValue` functions, I think only `StatsInsightData` can be stored in the cache? If that's the case, what do you think changing this type to `CacheKey: StatsInsightData`?

Reply:
That's better, updated!